lib/transaction/tracer/index.js

```diff
@@ -219,11 +213,7 @@ function _makeWrapped(tracer, handler, active, full) {
 }
 
 function getOriginal(fn) {
-  return fn && fn[ORIGINAL] ? fn[ORIGINAL] : fn
-}
-
-function getSegmentFromWrapped(fn) {
-  return fn && fn[SEGMENT] ? fn[SEGMENT] : null
+  return fn && fn[symbols.original] ? fn[symbols.original] : fn
```

Comment:
I guess people tend to forget that `&&` doesn't return bools and this ternary is just trying to be more explicit about it.

I'll use your suggestion instead.

Reply:
This accidentally broke everything, because `a && b` returns `b` if `a` is truthy but `b` is falsey, which is almost exactly what we don't want.

Instead, I went with ae443d5d9fc68866f5fdf366b8df0d77975c46da